scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -521,158 +520,178 @@
                         (map job->usage)
                         (reduce (partial merge-with +)))))))
 
+(defn category->pending-jobs->category->considerable-jobs
+  "Limit the pending jobs to considerable jobs based on usage and quota.
+   Further limit the considerable jobs to a maximum of num-considerable jobs."
+  [db category->pending-jobs user->quota user->usage num-considerable]
+  (log/debug "There are" (apply + (map count category->pending-jobs)) "pending jobs")
+  (log/debug "pending-jobs:" category->pending-jobs)
+  (let [filter-considerable-jobs (fn filter-considerable-jobs [jobs]
+                                   (->> jobs
+                                        (map #(d/entity db (:db/id %)))
```

Comment:
Ah, right, I see. Is it just pulling from the database to get updated information then?